src/apps/Compound/config.ts

```diff
@@ -22,7 +22,12 @@ export type TokenItem = {
 export const web3Provider = process.env.REACT_APP_WEB3_PROVIDER_URL || "";
 
 export const getTokenList = (network: Networks): Array<TokenItem> => {
+  if (network !== "rinkeby" && network !== "mainnet") {
```

Comment:
To be honest I would prefer if we clearly separate changes ... this PR should be for the Safe Connect app and here we make changes to the compound app. In general we should handle them more separately like a mono repo setup. We do something similar in the safe-modules repo where each subfolder represents a module and is independent from the others. Else we will run into situation where you update a lib and this always requires to update all Safe apps creating more changes then necessary.

Reply:
Agreed about handle this repo as a mono repo. What do you suggest to do right now, should I extract this change in a separated PR?

Reply:
hmmm yes I would suggest to have these changes in a different PR and then as a follow up issue (that we should create) we should convert this to mono repo. (Unless setting up the mono repo is fast then I would do that before)

Reply:
Ok, I'll split it up.
Moving to a Monorepo will take some time so it should be handled in a different issue.